app/src/main/java/com/telnyx/webrtc/sdk/ui/MainActivity.kt

```diff
@@ -182,6 +182,8 @@ class MainActivity : AppCompatActivity() {
             connectButtonPressed()
         }
         call_button_id.setOnClickListener {
+            //Change view / change button
+
```

Comment:
Should we leave these lines? I believe it might cause a conflict for the change I have pending too...